internal/metrics/remote_write_test.go

```diff
@@ -118,6 +171,7 @@ var _ = Describe("remote write", func() {
 			Expect(remoteWrite.IsEnabled()).To(BeTrue())
 			Expect(remoteWrite.IsRunning()).To(BeTrue())
 			Expect(remoteWrite.Config).To(Equal(configMessage2.Configuration.Metrics.Receiver))
+			Expect(listCaFiles()).To(HaveLen(0))
```

Comment:
Is `BeEmpty` more consistent with other tests than `HaveLen(0)`?

Reply:
I don't think it matters. But I do believe that BeEmpty is more commonly used for strings and HaveLen for collections.